packages/core/src/asciidoctor-core-api.js

```diff
@@ -851,7 +851,14 @@ AbstractNode.prototype.getNodeName = function () {
  * @memberof AbstractNode
  */
 AbstractNode.prototype.getAttributes = function () {
-  return fromHash(this.attributes)
```

Comment:
In theory we could do it in `fromHash` but in practice it has side effects.

Reply:
I changed my mind, I think it's more consistent to do it in `fromHash`. It also avoids an additional loop over the attributes.